include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -29,6 +27,9 @@
 
 #include <deal.II/dofs/dof_tools.h>
 
+#ifndef LETHE_GLSSHARPNS_H
+#  define LETHE_GLSSHARPNS_H
+
```

Comment:
Why was this moved after the #includes? Isn't it generally better to have it before the includes?

Reply:
I looked into it online and there are no general consensus about it. I think it should be better (with Bruno we looked in deal.II after I did the PR, and yes the includes are in the include guards). I did what was mostly done in Lethe, but I can do the opposite. It could be something in our in-house guidelines.